history/characters/400000_titan.txt

```diff
@@ -145,5 +145,9 @@
 	add_trait=brilliant_strategist add_trait=creature_titan_keeper add_trait=odyn_trait add_trait=stubborn add_trait=zealous
 	disallow_random_traits = yes
 	1.1.2={ birth=yes } #Must be born after Aman'thul
-	1094.12.6={ death=yes }
+	30.1.1={
+		death = {
+			death_reason = death_missing	# Not interactive anymore
```

Comment:
Rude of you to remove Valarjar, makes titan history inaccessible yeah?

Reply:
Zumbak did an oopsie. Forgot to mention it in the comment, sorry. 
Sadly, yes... Loken can take Odyn's role OR we can rename Odyn Havi, his human form.